### PR TITLE
No more codegen

### LINKED ISSUE #11: API validator image removed
The docker image that as being used for validation has been removed, the version that was being used is not available as a docker image, newer releases flag errors:

``` 
   1   (3%) : Property "license" is not expected to be here.
  28  (74%) : $ref must not be placed next to any other properties
   2   (5%) : Name collision detected between operation parameter and requestBody property
   1   (3%) : Path contains two or more consecutive path parameter references
   1   (3%) : "csv_result" property must have required property "$ref".
   1   (3%) : "content_disposition" property must not be valid.
   2   (5%) : Property "text/plain" is not expected to be here.
   1   (3%) : "created" property must have required property "$ref".
   1   (3%) : "see_other" property must have required property "$ref".
```

### PR DESCRIPTION
**Pull request by greenape**
Commits the generated code to the repo and removes code generation.

Closes #11

### CODE REVIEW

File: flowkit_ui_backend/impl/data_api_impl.py [hunk 1]
```diff
@@ -297,13 +309,15 @@ async def run_query(
...
+    tr = await get_temporal_resolution(
+        query_parameters.trid, token_model=token_model, pool=pool
+    )
 
     metadata = await get_metadata(query_parameters, tr, pool, token_model)
 
     # support getting mdids only
     if query_parameters.mdids_only is True:
-        return QueryResult(mdids=[m.mdid for m in metadata])
+        return QueryResult(mdids=[f"{m.mdid}" for m in metadata])
```
Comment (Thingus) [inline flowkit_ui_backend/impl/data_api_impl.py:320]: Does `m.mdid` kick out an int here by default?

**Review by Thingus — COMMENTED** [review]
Do we want to drop the 'do not edit this class manually' comment lines as a guard to those that come after?

File: flowkit_ui_backend/models/time_range.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    @validator("start_date")
+    def start_date_max_length(cls, value):
+        assert len(value) <= 19
+        return value
+
+    @validator("start_date")
+    def start_date_pattern(cls, value):
+        assert value is not None and re.match(
+            r"^([1-2][0-9]{3})((-(0[1-9]|1[0-2])(-(0[1-9]|[1-2][0-9]|3[0-1]))?(T(([0-1][0-9]|2[0-3]):([0-5][0-9])(:([0-5][0-9]))?))?)|(w(0[1-9]|[1-4][0-9]|5[0-3])))?$",
```
Comment (Thingus) [inline flowkit_ui_backend/models/time_range.py:58]: idk if we want to do this now, but this regex shows up a lot and is in a fairly hot path - is it worth compiling it on init?

File: tests/conftest.py [hunk 3]
```diff
@@ -31,8 +45,8 @@ async def fresh_pool(event_loop):
         An aiomysql pool connected to the db
     """
     pool = await aiomysql.create_pool(
-        host=os.getenv("CONTAINER_NAME_DB"),
-        port=int(os.getenv("DB_PORT_CONTAINER")),
+        host="localhost",
+        port=int(os.getenv("DB_PORT_HOST")),
```
Comment (Thingus) [inline tests/conftest.py:49]: We've used `os.getenv` here but `os.environ[]` elsewhere - is this significant?

File: tests/test_db.py [hunk 4]
```diff
@@ -44,42 +44,29 @@ def get_mock_pool():
 
 @pytest.mark.asyncio
 async def test_provision_db_subsequent_run(mocker, fresh_pool):
```
Comment (Thingus) [inline tests/test_db.py:46]: At a glance, can we drop the `mocker` class here?

File: flowkit_ui_backend/models/time_range.py (hunk 2 above)
Comment (greenape) [reply flowkit_ui_backend/models/time_range.py:58]: I think we probably want to replace it with something a bit less monstrous really - shall we add a todo instead? I feel like there has to be a better way to validate a date (like try and parse it as a date, for example).

File: tests/test_db.py (hunk 4 above)
Comment (greenape) [reply tests/test_db.py:46]: iirc this is to control order of fixtures

File: tests/conftest.py (hunk 3 above)
Comment (greenape) [reply tests/conftest.py:49]: I think this one should probably get environ really, cos we want to error if it is not set

Comment (greenape) [reply tests/conftest.py:49]:
```suggestion
        port=int(os.environ["DB_PORT_HOST"]),
```

File: flowkit_ui_backend/impl/data_api_impl.py (hunk 1 above)
Comment (greenape) [reply flowkit_ui_backend/impl/data_api_impl.py:320]: Yeah, it is an int in the db so gets typed

**Review by Thingus — APPROVED** [review]
I can run locally, so lgtm

**Pull request merged**